models/mpas_atm/model_mod.f90

```diff
@@ -5052,7 +5143,7 @@ subroutine convert_vert_distrib_state(state_handle, ens_size, location, quantity
 101 continue
 
      if (debug > 9 .and. do_output()) then
-        write(string2,'("zout_in_scaleheight:",F10.2)') zout
+        write(string2,'("zout_in_scaleheight:",F10.2)') zout(1)
```

Comment:
zout is a 1D array 'ens_size', printing out just one value is not necessarily the value with the problem.
This is in a fairly high debug level ... If they are digging that deep, they may want all the values.

Reply:
see earlier comment about having to loop if printing all values.  i think labeling it as member 1 is sufficient.

Reply:
There are many instances where it is not labelled as member 1 ... do you want me to change them?

Reply:
I'm just changing them at the moment thanks!

Reply:
changed to member 1 in debug message